.github/CODEOWNERS

```diff
@@ -325,8 +325,9 @@
 /core/platform/ @seanlip @DubeySandeep
 /core/templates/dev/head/App*.ts @ankita240796
 /core/templates/dev/head/app.constants.ts @ankita240796
+/core/templates/dev/head/app.constants.ajs.ts @ankita240796
```

Comment:
I have seen in [some writings](https://scotch.io/tutorials/get-started-with-ngupgrade-going-from-angularjs-to-angular) that they name the AJS equivalents as *.ajs.ts. However, if you want *.ajsconstants.ts, we can do that too. 
I personally favor *.ajs.ts since it is cleaner, also these are going to be removed no matter what the name. What do you think?

Reply:
Actually, if possible, I prefer .ajsconstants.ts since it is "dirtier" and it sticks out more. It serves as a good reminder that "this is temporary and we shouldn't be making major edits to it in the near-term". 

It's optional, though. You can make the call.